analyzer-core/src/main/java/ffb/analyzer/core/queue/Task.java

```diff
@@ -0,0 +1,43 @@
+package ffb.analyzer.core.queue;
+
+import java.util.function.Consumer;
+
+public class Task<T> {
+    public int id;
+
+    private T objectToPerformTaskOn;
+    private Consumer<T> actionToPerform;
+
```

Comment:
There should a constructor here rather than setting `actionToPerform` or `objectToPerformTaskOn` via accessors.

Reply:
Updated together yesterday